src/fastcs_eiger/eiger_parameter.py

```diff
@@ -53,3 +54,16 @@ def fastcs_datatype(self) -> DataType:
 
 def key_to_attribute_name(key: str):
     return key.replace("/", "_")
+
+
+def minimum_to_precision(value: float | None) -> int:
+    if value is not None:
+        value_as_str = str(value)
+        for separator in [".", "e"]:
+            if separator in value_as_str:
+                return (
+                    len(value_as_str.split(separator)[1])
+                    if separator == "."
+                    else abs(int(value_as_str.split(separator)[1]))
+                )
```

Comment:
Could: Something like this seems cleaner

```suggestion
        if "." in value_as_str:
             return len(value_as_str.split(separator)[1])
        elif "e" in value_as_str:
             return abs(int(value_as_str.split(separator)[1]))
```

Reply:
So there is, my bad. Thanks!